### ISSUE TITLE
dependencies for mops faulty

### ISSUE THREAD

**Issue by letmejustputthishere** [NatLabs/serde#2]
i cannot use `serde` for my project, this [dapp](https://m7sm4-2iaaa-aaaab-qabra-cai.raw.ic0.app/?tag=11814075) is not working locally with `serde` as a mops dependency. it works with `vessel` though. i reuploaded it to mops with this `mops.toml` under the name `motoko-serde` and now it works.

```
[package]
name = "motoko-serde"
version = "0.1.4"
description = "A serialisation and deserialisation library for Motoko."
repository = "https://github.com/NatLabs/serde"

[dependencies]
base = "https://github.com/dfinity/motoko-base#moc-0.7.4"
json = "https://github.com/aviate-labs/json.mo#v0.2.0"
itertools = "https://github.com/NatLabs/Itertools#master"
candid = "https://github.com/gekctek/motoko_candid#main"
"xtended-numbers" = "https://github.com/gekctek/motoko_numbers#v1.0.0"
"parser-combinators" = "https://github.com/aviate-labs/parser-combinators.mo#master"
```

please consider upgrading the dependencies

**Comment by tomijaga** [comment]
Thanks for reporting this issue.
I didn't get a notification for the issues in this repo, 
but I will look into it now and update the dependencies.

**Comment by tomijaga** [comment]
@letmejustputthishere I've updated the dependencies and published a new version to mops `0.1.2`
